checkov/terraform/checks/resource/oci/SecurityListIngressStateless.py

```diff
@@ -0,0 +1,27 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class SecurityListIngressStateless(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure VCN inbound security lists are stateless"
+        id = "CKV_OCI_17"
+        supported_resources = ['oci_core_security_list']
+        categories = [CheckCategories.GENERAL_SECURITY]
```

Comment:
```suggestion
        categories = [CheckCategories.NETWORKING]
```
fits here better, also other ingress checks have that category.